src/React.Core/ReactSiteConfiguration.cs

```diff
@@ -344,5 +345,20 @@ public IReactSiteConfiguration SetScriptNonceProvider(Func<string> provider)
 			ScriptNonceProvider = provider;
 			return this;
 		}
+
+		/// <summary>
+		/// Gets or sets whether HTML extensions methods writes its output directly to ViewContext.Writer 
+		/// </summary>
+		public bool UseViewContextWriter { get; set; }
+
+		/// <summary>
+		/// Gets or sets whether using HTML extensions methods writes its output directly to ViewContext.Writer
+		/// </summary>
+		/// <returns>The configuration, for chaining</returns>
+		public IReactSiteConfiguration SetUseViewContextWriter(bool useViewContextWriter)
+		{
+			UseViewContextWriter = UseViewContextWriter;
```

Comment:
`UseViewContextWriter = useViewContextWriter;` not `UseViewContextWriter = UseViewContextWriter;`